### ISSUE TITLE
I’m apparently not skilled enough

### ISSUE THREAD

**Issue by rockstarjsw** [NL647/DuinoCoin-balance-Home-Assistant#2]
Hi all, I’m trying to learn things in HASS. I don’t know how to add this to my server. At the very beginning it states download repo… does that just mean the 3 files? Also I was looking through the code and it seems all I need to input is my user name and the file directory for the icon… there are also 2 folders I don’t have in my config, the local folder  and the "custom_components" folder… is this something I can simply add or am I missing something?

Thank you in advance, and sorry if I’m too much a noob.

**Comment by rockstarjsw** [comment]
<img width="775" alt="Screen Shot 2021-12-04 at 9 02 08 PM" src="https://user-images.githubusercontent.com/48425667/144734383-0c543b74-7ca1-48f5-94d1-ee70ece1d6f6.png">
I can't get it to load with HACS either. I have everything up to date...

**Comment by thomasgregg** [comment]
A HACS compatible repo would be great. Would also like to use it with HACS.

**Comment by rockstarjsw** [comment]
I ended up getting it working... mostly. I just created the folders that the publisher assumed I had (maybe most people do) input the files, changed username to my name and force restarted HASS. HASS didn't want to, gave me an error so I had to reset power to the device, came back online with this working. Scary, wish it integrated easier, but alas it works... now my only issue is getting the Duino-Coin image to show... I couldn't so I changed it to the dollar sign icon.
